### PR TITLE
Updated jenkins file to run smoke test

### PR DESCRIPTION
**Pull request by dmareddx**
Added a new stage "Run SmokeTest" to jenkins file.
In this stage it will pull all dependency binaries from sdo components 
and run the smoke test configured in sdo-test repository.
Smoke test logs will be archived under SmokeTest_Log file in jenkins

Signed-off-by: Mareddy, Deepthi <deepthix.mareddy@intel.com>

### CODE REVIEW

File: Jenkinsfile.yml [hunk 1]
```diff
@@ -55,5 +58,77 @@ node('ccode'){
...
+    try{
+      stage('Run SmokeTest'){
+        checkout([$class: 'GitSCM', branches: [[name: '*/master']], extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'sdo-test']], userRemoteConfigs: [[credentialsId: 'sdo-automationgithubtoken', url: 'https://github.com/secure-device-onboard-ci/sdo-test']]])
+        sh '''
+          mkdir -p sdo-test/binaries/pri
+          mkdir -p sdo-test/binaries/supply-chain-tools
+          mkdir -p sdo-test/binaries/rendezvous-service
+          mkdir -p sdo-test/binaries/client-sdk/x86_ecdsa256_sct_bin
+          mkdir -p sdo-test/binaries/client-sdk/x86_ecdsa384_sct_bin
+          mkdir -p sdo-test/binaries/client-sdk/x86_epid_sct_bin
```
Comment (trbehera) [inline Jenkinsfile.yml:71]: Please remove EPID related commands. EPID support is being removed as part of PR #3

File: Jenkinsfile.yml [hunk 2]
```diff
@@ -55,5 +58,77 @@ node('ccode'){
...
+          tar -xvzf sdo-test/binaries/supply-chain-tools/demo.tar.gz -C sdo-test/binaries/supply-chain-tools/ --strip=1
+          tar -xvzf sdo-test/binaries/rendezvous-service/demo.tar.gz -C sdo-test/binaries/rendezvous-service/ --strip=1
+        '''
+        print "Copy client-sdk artifacts to sdo-test/binaries folder"
+        sh '''
+          tar -xvzf client-sdk/x86_ecdsa256_c_sct_bin.tar.gz -C sdo-test/binaries/client-sdk/x86_ecdsa256_sct_bin/ --strip=1
+          cp sdo-test/testFiles/keys/ecdsa256privkey.dat sdo-test/binaries/client-sdk/x86_ecdsa256_sct_bin/data/
+          tar -xvzf client-sdk/x86_ecdsa384_c_sct_bin.tar.gz -C sdo-test/binaries/client-sdk/x86_ecdsa384_sct_bin/ --strip=1
+          cp sdo-test/testFiles/keys/ecdsa384privkey.dat sdo-test/binaries/client-sdk/x86_ecdsa384_sct_bin/data/
+          tar -xvzf client-sdk/x86_epid_c_sct_bin.tar.gz -C sdo-test/binaries/client-sdk/x86_epid_sct_bin/ --strip=1
```
Comment (trbehera) [inline Jenkinsfile.yml:89]: Please remove EPID related commands.

**Comment by trbehera** [comment]
@dmareddx @AdithyaBaglody  Do we need this PR after PR #3 has been merged?

**Comment by dmareddx** [comment]
> @dmareddx @AdithyaBaglody Do we need this PR after PR #3 has been merged?

**Comment by dmareddx** [comment]
This is a duplicate of PR#11 and this can be closed

**Pull request closed without merging**